### PR TITLE
3941 - Datagrid Refresh Column

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR adds the ability to update values of a specific column on Datagrid.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/3941

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/datagrid/test-update-column.html
- Click on the Load Column

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Review by tmcconechy — CHANGES_REQUESTED** [review]
I think this is actually updating every cell by calling updateCell in a loop? But try this:

1. change data to
```
        const dataset2 = [{
            productId: 1,
            productName: 'Bumpers 1',
            activity: 'Assemble Paint 1',
            quantity: 1
          }, {
            productId: 2,
            productName: 'Wheels 2',
            activity: 'Inspect and Repair 2',
            quantity: 2
          }, {
            productId: 3,
            productName: 'Suspension 3',
            activity: 'Cleaning 3',
            quantity: 1
          }
        ];
 ```
 
 1. hit update/load column
 1. would expect the productName column to not refresh only the one column should?

File: app/views/components/datagrid/test-update-column.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        <div class="title">
+          Compressors
+        </div>
+        <div class="buttonset">
+
+          <label class="audible" for="common-toolbar-searchfield">Keyword Search</label>
+          <input id="common-toolbar-searchfield" name="common-toolbar-searchfield" class="searchfield" />
+
+          <button type="button" class="btn" id="load-column">
+            <span>Load Column</span>
```
Comment (tmcconechy) [inline app/views/components/datagrid/test-update-column.html:13]:
```suggestion
            <span>Update Activity Column</span>
```

**Review by tmcconechy — DISMISSED** [review]
Looks like its working as I expect now. Maybe you can add a functional test for this new API?

**Comment by tjamesallen15** [comment]
Sure @tmcconechy

**Review by tmcconechy — APPROVED** [review]
Nice test! Thanks!

**Review by deep7102 — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**